website/_docs/cmd.clock.markdown

```diff
@@ -8,8 +8,13 @@ permalink: docs/cmd/clock.html
 
 Returns the current clock value for a watched root.
 
-Be careful how you interpret this value; it returns the instantaneous value of
-the clock, and may have changed by the time you intend to act upon it.
+If a sync_timeout option is supplied, the command blocks for the given
+number of microseconds waiting for all pending changes to sync.  The
```

Comment:
the timeout is measured in milliseconds.

The description of what this does is a little inaccurate.  It's probably a good idea to link to http://facebook.github.io/watchman/docs/cookies.html#how-cookies-work but summarize it here as:

sync_timeout specifies the number of milliseconds that you want to wait to observe a synchronization cookie.  The synchronization cookie is created at the start of your `clock` call and, once the cookie is observed, means that the clock value returned by this command is at least as current as the time of your `clock` call.

If no sync_timeout is specified, the returned clock value is the instantaneous value of the clock associated with the watched root, and may be almost immediately invalidated if there are any filesystem notifications that are yet to be processed.